src/utils/isWeb.ts

```diff
@@ -1,3 +1,5 @@
-export default typeof window !== 'undefined' &&
-  typeof window.HTMLElement !== 'undefined' &&
-  typeof document !== 'undefined';
+const UNDEFINED = 'undefined';
```

Comment:
Would you save more bytes by using `let`? 😆

```suggestion
let UNDEFINED = 'undefined';
```

Reply:
I have been thinking about it change all const to let

`1:5  error  'UNDEFINED' is never reassigned. Use 'const' instead  prefer-const`